src/particles/elements/ExactSbend.H

```diff
@@ -0,0 +1,173 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_EXACTSBEND_H
+#define IMPACTX_EXACTSBEND_H
+
+#include "particles/ImpactXParticleContainer.H"
+#include "mixin/beamoptic.H"
+#include "mixin/thick.H"
+#include "mixin/nofinalize.H"
+
+#include <AMReX_Extension.H>
+#include <AMReX_REAL.H>
+#include <AMReX_Print.H>      // for PrintToFile
+
+#include <cmath>
+
+
+namespace impactx
+{
+    struct ExactSbend
```

Comment:
Do you want to cite the ref papers here?

Reply:
Done.